src/main/java/de/hysky/skyblocker/config/configs/ChatConfig.java

```diff
@@ -61,7 +61,5 @@ public class ChatConfig {
     public static class ChatRuleConfig {
         @SerialEntry
         public int announcementLength = 60;
```

Comment:
Just deprecate the config options to prevent future conflicts.
```suggestion
        public int announcementLength = 60;

        @Deprecated
        @SerialEntry
        public int announcementScale = 3;
```